### DISCUSSION TITLE
Looking for Data Visualization Solutions in Latest DKAN Version

### DISCUSSION THREAD

**Discussion by mitthuzyxware** [GetDKAN/dkan#4028] [Migration help]
Hi,

I have been working with DKAN in Drupal 10 and have enabled the DKAN JS Frontend for a decoupled front end. It is working fine. However, in DKAN 2.x, I did not find any React components or Drupal modules that can allow us to visualize datasets in graphs, charts, maps, and other options. Do we have such capabilities in the latest version of DKAN?

**Comment by dafeder** [comment]
@mitthuzyxware not out of the box. What DKAN provides toward this is the ability to import datasets into a datastore which gives you a public API to do SQL-like queries. You can then use this with your graphing library of choice on the front end.

**Reply by mitthuzyxware** [reply]
Thanks @dafeder ,
I am aware of it and planned to do same. I just wanted to make sure if there is any existing components available for these visualization options like in Drupal 7 DKAN.
I will create these options with the help of react components and custom module.
